src/components/TipCard/TipCard.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './TipCard.css'
```

Comment:
I don't believe we need this import in this file, as this is the TipCard component and we aren't rendering another component inside of TipCard itself.